deployment/src/test/java/io/quarkiverse/renarde/test/UriBuilderWithPrefixTest.java

```diff
@@ -0,0 +1,94 @@
+package io.quarkiverse.renarde.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Duration;
+import java.util.List;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.QueryParam;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkiverse.renarde.Controller;
+import io.quarkus.mailer.Mail;
+import io.quarkus.mailer.MailTemplate;
+import io.quarkus.mailer.MockMailbox;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class UriBuilderWithPrefixTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(TestService.class)
+                    .addAsResource(new StringAsset("quarkus.http.root-path=/support"), "application.properties")
+                    .addAsResource(new StringAsset(
+                            "{uri:MyController.endpoint()}\n{uriabs:MyController.endpoint()}\n{uri:MyController.endpoint('param')}\n{uriabs:MyController.endpoint('param')}"),
+                            "templates/MyController/email.html"));
+
+    public static class MyController extends Controller {
+
+        @CheckedTemplate
+        public static class Templates {
+            public static native MailTemplate.MailTemplateInstance email();
+        }
+
+        @Path("/endpoint-url")
+        public void endpoint(@QueryParam("t") String testParam) {
+        }
+    }
+
+    @Inject
+    TestService testService;
+
+    @Inject
+    MockMailbox mockMailbox;
+
+    @Test
+    void testDefaultUriBuilderIsUsed() {
+        testService.sendQuteEmail();
+
+        var email = "none@icann.org";
```

Comment:
I think the preferred domain is `@example.com` https://en.wikipedia.org/wiki/Example.com